WcaOnRails/config/environments/test.rb

```diff
@@ -43,7 +43,7 @@
   config.active_support.deprecation = :stderr
 
   # Raises error for missing translations
-  config.action_view.raise_on_missing_translations = true
+  # config.action_view.raise_on_missing_translations = true
```

Comment:
@jonatanklosko I just enabled i18n fallbacks in tests too.
When we set up i18n we thought raising error could help us catch issues with locales, but tbh for the past 4 years it's been used I believe it never happened and only caused us issues when adding new keys in the navigation/homepage...

Not that the setting is useless, but we already have an i18n spec that make sure that all key used exist in English, so we are sure to always have at least that.

What do you think about the change?

Reply:
Sounds good to me!